src/Scientist/Internals/ExperimentInstance.cs

```diff
@@ -60,8 +61,14 @@ internal ExperimentInstance(string name, NamedBehavior control, NamedBehavior ca
             }
 
             // Randomize ordering...
+            NamedBehavior[] orderedBehaviors;
+            lock (_randomLock)
```

Comment:
Not a big deal, but since `_random` is a private instance and not shared anywhere, we could have just locked on `_random` itself.